ocaml/xapi/xapi_session.ml

```diff
@@ -766,6 +775,7 @@ let login_with_password ~__context ~uname ~pwd ~version:_ ~originator =
             ~auth_user_sid:"" ~auth_user_name:uname ~rbac_permissions
             ~db_ref:None ~client_certificate:true
       | None -> (
+          let () = assert_username uname in
```

Comment:
Would `assert_username_valid uname ;` be better?

Reply:
It's a matetr of style, both are fine

Reply:
@minglumlu Thanks for the reminder.  I have updated and fixed format issue, please review again.